### DISCUSSION TITLE
Realistic Skin

### DISCUSSION THREAD

**Discussion by williamsblood** [lllyasviel/Fooocus#671] [General]
Hello, I've been using the new Realistic Vision SDXL 2.0 checkpoint to generate some photo-realistic images. While the face portraits often turn out perfect, I'm encountering some issues with full-body shots, particularly with the skin texture on arms or chest areas appearing somewhat plastic-like. I've already adjusted the sharpness setting to 10, but still, some full-body results are not as realistic as I'd like. Does anyone have suggestions on how to optimize the Fooocus settings to improve this? Thank you.

![00001-1275672473](https://github.com/lllyasviel/Fooocus/assets/144069983/d10c78e8-8555-4048-95e5-3dfb6c9aa207)
![image - 2023-10-13T125815 193](https://github.com/lllyasviel/Fooocus/assets/144069983/0463c3ed-ce9f-4629-9c28-7fc1347e91ad)

**Comment by xhoxye** [comment]
Try using the related SD1.5 to act as a refiner to enhance the fidelity of the skin

**Comment by lllyasviel** [comment]
try upscale

maybe related
https://github.com/lllyasviel/Fooocus/discussions/679
